cluster/ci/config/prow/config.yaml

```diff
@@ -442,6 +446,7 @@ presubmits:
     # TODO: Remove once the job is stable
     skip_report: true
     branches:
+    - cli-pre-rebase
```

Comment:
You don't need this since it's still an experimental job; remove

Reply:
done, thanks